samples: nrf9160: modem_shell: bluetooth support (#7876)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

nRF9160 samples
---------------

* :ref:`modem_shell_application` sample:

  * Added:

>     * nRF9160 DK overlays for enabling BT support. When running this configuration, BT scanning and advertising can be made with ``bt`` command.

Reviewer: ```suggestion
    * nRF9160 DK overlays for enabling BT support.
      When running this configuration, you can perform BT scanning and advertising using the ``bt`` command.
```

nRF9160 samples
---------------

* :ref:`modem_shell_application` sample:

  * Added:

>     * nRF 9160 DK overlays for enabling BT support. When running this configuration, BT scanning and advertising can be mabe with ``bt`` command.

Reviewer 2: As a general comment, `nRF9160` should be written without a space. The same change should be done for the commit message.

Typo: "mabe"

Author: Fixed.
